src/dmangle.d

```diff
@@ -88,6 +94,9 @@ private immutable char[TMAX] mangleChar =
     //              X   // variadic T t...)
     //              Y   // variadic T t,...)
     //              Z   // not variadic, end of parameters
+    //              %   // Type backward reference
+    //              ?   // symbol backward reference
+    //              $   // identifier backward reference
```

Comment:
I would go on the assumption that they can never be reused.  As once you change it, you can no longer debug old programs.